workshops/bootstrap_site/README.md

```diff
@@ -0,0 +1,260 @@
+# Bootstrap Site
+
+In this workshop, we're going to build a similar website to what we started in [Personal Website](../personal_website/README.md). We'll be introducing a tool called [Bootstrap](https://v4-alpha.getbootstrap.com/), which is a set of pre-written CSS that makes it super easy to build complex sites.
+
+Hundreds of thousands of websites, including http://www.nba.com/, https://www.lyft.com/, and https://www.nasa.gov/, are built on Bootstrap. Check out https://expo.getbootstrap.com/ for more examples.
+
+Let's get started.
+
+Please save after each section and refresh Live Preview!
+
+## 1) Set-up
+
+### Creating Files
+
+- Create new folder (right-click sidebar and choose new folder, name it `bootstrap_site`)
+- Create `index.html` (right-click `bootstrap_site` folder and choose new file, name it `index.html`)
+
+### Priming `index.html`
+
+- Double-click `index.html` to open, and type in the following base template:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+  </head>
+  <body>
+  </body>
+</html>
+```
+
+- In this workshop we will be using Bootstrap, which is a CSS framework made by Twitter, to make beautiful websites extremely easy to create.
+  Since bootstrap is external to our `index.html`, we must include it.
```

Comment:
Maybe say this instead:

```md
Just like how we can link to CSS files we've created using the `<link>` tag, we can link to CSS files others have created – just instead of including just the filename (like `main.css`), we put the full URL to the CSS file (in this case, `https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.2/css/bootstrap.min.css`).
```